utils/lib/connection_database.sql

```diff
@@ -252,3 +262,42 @@ CREATE TABLE constant_sources(
     FOREIGN KEY(vcc_track_pkey) REFERENCES track(pkey),
     FOREIGN KEY(gnd_track_pkey) REFERENCES track(pkey)
 );
+
+-- Grid location map.
+CREATE TABLE grid_loc_map(
+    phy_tile_pkey INT,
+    vpr_tile_pkey INT,
+    FOREIGN KEY(phy_tile_pkey) REFERENCES phy_tile(pkey),
+    FOREIGN KEY(vpr_tile_pkey) REFERENCES tile(pkey)
+);
+
+-- Tile type map.
+-- Maps physical tiles to VPR tiles. Used for CLB splitting
```

Comment:
This map is redudaent with the grid_loc_map?

Reply:
It maps tile types instead of their instances in the grid.

Reply:
But there is a never a case to use this map?  When would you use it?

Reply:
You are probably right. It is not used now but I can't be sure whether it is not going to be needed at some point of the flow.

Reply:
I guess what I'm saying is that this information is better suited to a column in another table, for example the "site_remap_pkey" in the tile type table seems better than this table.

Reply:
AFAIU "site_remap_pkey" is another thing. Here I want to store information about tile type split. For example that the CLBLM_L splits into SLICEM and SLICEL. But now as I think of it I am not sure whether the information is needed anywhere in the flow.